clang/test/dpct/asm/cp.cu

```diff
@@ -0,0 +1,70 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/cp %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/cp/cp.dp.cpp
+// RUN: %if build_lit %{icpx -c -fsycl %T/cp/cp.dp.cpp -o %T/cp/cp.dp.o %}
+
+// clang-format off
+#include <cstdint>
+#include <cstdint>
+#include <cuda_runtime.h>
+
+// CHECK: inline void cp_async4(void *smem_ptr, const void *glob_ptr) {
+// CHECK-NEXT:  const int BYTES = 16;
+// CHECK-NEXT:  auto smem = smem_ptr;
+// CHECK-NEXT:  /*
+// CHECK-NEXT:  DPCT1137:{{[0-9]+}}: Synchronous copy operations are used on the SYCL side for migration of PTX "cp.async" and may cause performance issues. You may need to adjust.
```

Comment:
ASM instruction "cp.async" is asynchronous copy,  current it is migrated to synchronous copy operation.  You may need to adjust the code to tune the performance.